content/articles/building-a-lyrics-search-app-using-vanilla-javascript/index.md

```diff
@@ -0,0 +1,490 @@
+
+## Building a Lyrics Search App Using Vanilla JavaScript
+
+![hero image](/building-lyrics-search-app-using-vanilla-javascript/hero.png)
+
+Not knowing the lyrics of a song, is a problem most song-lovers encounter. In this article, you will be creating a platform where users can search for lyrics by entering the artist name or title of the song.
+
+You will be creating the search app using HTML5, CSS3, Vanilla JavaScript, async-await with fetch method, OVH API, and ECMAScript 2015 (ES6) features like arrow functions.
+
+### Prerequisites
+
+To follow along with this tutorial, you need the following:
+
+- A code editor (like Visual Studio Code).
+- Basic knowledge of HTML5, CSS, and JavaScript.
+- A laptop with an internet connection.
+
+### Structuring the lyrics search app with HTML5
+To begin, create a folder named `lyrics-app`, then you need to create three files in the folder: `lyrics.html`, `lyrics.css`, and `lyrics.js`.
+
+In the `lyrics.html` file, copy the code below and paste it in your code editor to create the structure for the lyrics search app.
```

Comment:
` copy the code below and paste`

Instead of saying this, explain what is being done here and why are we doing it.

Reply:
Do the same for other code snippets as well.